### PR TITLE
Filterable fellow list

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #1215 

https://foundation-mofostaging-pr-1248.herokuapp.com/fellowships/directory/

- updated the main fellowship directory to filterable list
- removed `/fellowships/directory/archive` page
- redirected ` /fellowships/directory/archive` to  /fellowships/directory`

### CODE REVIEW

File: source/js/components/fellow-list/fellow-list.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    super(props);
+
+    this.state = {
+      fellows: null
+    };
+  }
+
+  fetchFellows(query) {
+    Object.assign(query, {
+      'profile_type': `fellow`,
```
Comment (gvn) [inline source/js/components/fellow-list/fellow-list.jsx]: Shouldn't need single quotes around the property names...

File: source/js/components/fellow-list/fellow-list.jsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    if ( fellow.twitter ) {
+      links.twitter = fellow.twitter;
+    }
+
+    if ( fellow.linkedin ) {
+      links.linkedIn = fellow.linkedin;
+    }
+
+    let metadata = {
+      'internet_health_issues': issues,
```
Comment (gvn) [inline source/js/components/fellow-list/fellow-list.jsx]: No need for single quote

File: source/js/components/fellow-list/fellow-list.jsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+      links: links,
+      name: fellow.name,
+      role: `${fellow.profile_type}, ${fellow.program_year}`,
+      image: fellow.thumbnail,
+      location: fellow.location,
+      affiliations: [], // don't show affiliations meta for now
+      'custom_link': { text: `See work`, link: `https://${this.props.env.PULSE_DOMAIN}/profile/${fellow.profile_id}` }
```
Comment (gvn) [inline source/js/components/fellow-list/fellow-list.jsx]: No need for single quote

File: source/js/components/fellow-list/fellow-list.jsx (hunk 1 above)
Comment (mmmavis) [reply source/js/components/fellow-list/fellow-list.jsx]: I remember linter requires quotes... but I'll try it again without quotes and see if linter flags it

Comment (mmmavis) [reply source/js/components/fellow-list/fellow-list.jsx]: k so linter does want us to keep the quotes...

**Comment by mmmavis** [comment]
mergingggggg 🎉

**Pull request merged**